### ISSUE TITLE
<div> component error

### ISSUE THREAD

**Issue by JinHoSo** [johniak/react-web-gifted-chat#12]
#### Issue Description

When I type something this error occurs at the moment

```
The above error occurred in the <div> component:
    in div (created by TouchableOpacity)
    in TouchableOpacity (created by Send)
    in Send (created by InputToolbar)
    in div (created by View)
    in View (created by InputToolbar)
    in div (created by View)
    in View (created by InputToolbar)
    in InputToolbar (created by GiftedChat)
    in div (created by View)
    in View (created by GiftedChat)
    in GiftedChat (at Message.tsx:165)
    in Message (created by WithStyles(Message))
    in WithStyles(Message) (created by Context.Consumer)
    in Connect(WithStyles(Message)) (at Chat.tsx:135)
    in div (at Chat.tsx:134)
    in div (created by SplitterPane)
    in SplitterPane (created by Splitter)
    in div (created by Splitter)
    in Splitter (at Chat.tsx:125)
    in div (at Chat.tsx:124)
    in Chat (created by WithStyles(Chat))
    in WithStyles(Chat) (created by Context.Consumer)
    in Connect(WithStyles(Chat)) (created by Route)
    in Route (at App.tsx:40)
    in Switch (at App.tsx:38)
    in main (at App.tsx:37)
    in div (at App.tsx:36)
    in App (created by WithStyles(App))
    in WithStyles(App) (at src/index.tsx:23)
    in MuiThemeProviderOld (at src/index.tsx:21)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Context.Consumer)
    in ConnectedRouterWithContext (created by Context.Consumer)
    in Connect(ConnectedRouterWithContext) (at src/index.tsx:20)
    in PersistGate (at src/index.tsx:19)
    in Provider (at src/index.tsx:18)
```

```
Uncaught TypeError: Failed to set an indexed property on 'CSSStyleDeclaration': Index property setter is not supported.
    at setValueForStyles (react-dom.development.js:6713)
    at setInitialDOMProperties (react-dom.development.js:7855)
    at setInitialProperties (react-dom.development.js:8084)
    at finalizeInitialChildren (react-dom.development.js:9158)
    at completeWork (react-dom.development.js:17644)
    at completeUnitOfWork (react-dom.development.js:20108)
    at performUnitOfWork (react-dom.development.js:20315)
    at workLoop (react-dom.development.js:20326)
    at renderRoot (react-dom.development.js:20406)
    at performWorkOnRoot (react-dom.development.js:21363)
    at performWork (react-dom.development.js:21273)
    at flushInteractiveUpdates$1 (react-dom.development.js:21540)
    at batchedUpdates (react-dom.development.js:2260)
    at dispatchEvent (react-dom.development.js:5110)
    at react-dom.development.js:21526
    at Object.unstable_runWithPriority (scheduler.development.js:255)
    at interactiveUpdates$1 (react-dom.development.js:21525)
    at interactiveUpdates (react-dom.development.js:2268)
    at dispatchInteractiveEvent (react-dom.development.js:5086)
```
[FILL THIS OUT]

#### Steps to Reproduce / Code Snippets

[FILL THIS OUT]

#### Expected Results

[FILL THIS OUT]

#### Additional Information

<img width="1680" alt="스크린샷 2019-05-11 오후 6 03 28" src="https://user-images.githubusercontent.com/31269303/57567646-6a6b5680-7417-11e9-8e7a-1b676ae4ea97.png">

<img width="1679" alt="스크린샷 2019-05-11 오후 6 03 39" src="https://user-images.githubusercontent.com/31269303/57567645-6a6b5680-7417-11e9-8eaa-1697edffd724.png">

* Nodejs version: 
* React version: 16.7.0
* react-web-gifted-chat version: 0.6.4
* Browser(s) (Chrome, etc.): chrome 74.0.3729.131

**Comment by s-turcheniuk** [comment]
same issue

**Comment by s-turcheniuk** [comment]
'Send' component receives an incorrect style prop which causes this issue. A quick fix is to add 'renderSend' prop to GiftedChat component.

**Comment by johniak** [comment]
Fixed in v0.6.6
